source/lib/cdk-textract-stack.ts

```diff
@@ -973,6 +1191,15 @@ export class CdkTextractStack extends cdk.Stack {
       }
     );
 
+    // add kendra index id to lambda environment in case of DUS+Kendra mode
+    if(props.enableKendra && createKendraIndexCustomResource!=null){
+      apiProcessor.addEnvironment("KENDRA_INDEX_ID",createKendraIndexCustomResource.getAtt('KendraIndexId').toString())
+      apiProcessor.addEnvironment("KENDRA_ROLE_ARN",kendraRoleArn)
+      jobResultProcessor.addEnvironment("KENDRA_INDEX_ID",createKendraIndexCustomResource.getAtt('KendraIndexId').toString())
+      jobResultProcessor.addEnvironment("KENDRA_ROLE_ARN",kendraRoleArn)
+      syncProcessor.addEnvironment("KENDRA_INDEX_ID",createKendraIndexCustomResource.getAtt('KendraIndexId').toString())
+      syncProcessor.addEnvironment("KENDRA_ROLE_ARN",kendraRoleArn)
+    }
```

Comment:
You also need to add Kendra permissions to index, query  and delete for some of these lambda/ APIs